test/utils/runners.go

```diff
@@ -193,6 +202,22 @@ func RunDeployment(config DeploymentConfig) error {
 	return config.start()
 }
 
+func (config *DeploymentConfig) Run() error {
+	return RunDeployment(*config)
+}
+
+func (config *DeploymentConfig) GetName() string {
```

Comment:
I think that you don't need to redefine GetName() and GetNamespace() if you are just calling the same method from base object.